cmd/keytransparency-client/cmd/post.go

```diff
@@ -35,12 +33,12 @@ var (
 
 // postCmd represents the post command
 var postCmd = &cobra.Command{
-	Use:   "post [user email] -d {profile}",
+	Use:   "post [user email] [app] -d {key data}",
 	Short: "Update the account with the given profile",
 	Long: `Post replaces the current key-set with the provided key-set, 
 and verifies that both the previous and current key-sets are accurate. eg:
 
-./keytransparency-client post foobar@example.com -d '{"app1": "dGVzdA==", "app2": "dGVzdA=="}'
+./keytransparency-client post foobar@example.com app1 -d "01020304"
```

Comment:
What's the motivation of this change? And can we still pass multiple apps in one command as it used to be before?